core/src/components/toast/toast.ios.scss

```diff
@@ -22,6 +22,10 @@
   display: block;
   position: absolute;
 
+  font-weight: 600;
```

Comment:
Can we move this to a sass variable in `toast.ios.vars.scss`?

Reply:
@liamdebeasi What about chaning the font-weight of the header to 600 too?
It does not look good if the header is thinner than everything else.